vmm/Cargo.toml

```diff
@@ -5,10 +5,8 @@ authors = ["The Cloud Hypervisor Authors"]
 edition = "2018"
 
 [features]
-default = []
+default = ["pci", "vfio-ioctls"]
```

Comment:
This and the other similar cases should just be made dependencies.

Reply:
I've updated the PR to address this comment. PTAL.